pkgs/development/python-modules/python-redis-lock/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, redis
+, pytest
+, process-tests
+, pkgs
+}:
+
+buildPythonPackage rec {
+  pname = "python-redis-lock";
+  version = "3.3.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "5316d473ce6ce86a774b9f9c110d84c3a9bd1a2abfda5d99e9c0c8a872a8e6d6";
+  };
+
+  checkInputs = [ pytest process-tests pkgs.redis ];
+
+  checkPhase = ''
+    pytest tests/
+  '';
+
+  propagatedBuildInputs = [ redis ];
```

```suggestion
  propagatedBuildInputs = [ redis ] 
  ++ stdenv.lib.optional withDjango django_redis;
```

How you implement https://github.com/NixOS/nixpkgs/pull/62668/files#r291694999